pylint_plugins/print_function.py

```diff
@@ -0,0 +1,44 @@
+import os
+
+import astroid
+from pylint.interfaces import IAstroidChecker
+from pylint.checkers import BaseChecker
+
+
+IGNORE_DIRS = list(
+    map(
+        os.path.abspath,
+        [
+            "dev",
+            # TODO: Remove (or replace) print statements in tests
+            "tests",
```

Comment:
I'll address this in a follow-up or let contributors fix them.

Reply:
I think we can simply remove print statements in tests because they are not that helpful and should not affect test results.